src/main/frontend/components/reference.cljs

```diff
@@ -32,6 +67,9 @@
           scheduled-or-deadlines (if journal?
                                    (db/get-date-scheduled-or-deadlines (string/capitalize page-name))
                                    nil)
+          references (remove #{page-name} (distinct (flatten (map filtering/get-nested-block-references (flatten (map val ref-blocks))))))
```

Comment:
Consider using thread macros `->>` to make it easier to read :)
https://clojuredocs.org/clojure.core/-%3E%3E